proposals/lip-0055.md

```diff
@@ -105,16 +129,45 @@ The full processing of a block is organized as follows.
 5. **Assets verification**: Each module verifies the respective entry in the block assets. If any check fails, the block is discarded and has no further effect.
 6. **Block forwarding**: After the initial checks, the full block is forwarded to a subset of peers.
```

Comment:
This step is not shown in the diagram anymore.

Reply:
added

Reply:
I still do not see the block forwarding in the diagram.

Reply:
fixed